src/store/settingsSlice.ts

```diff
@@ -24,7 +24,7 @@ const initialState: SettingsState = {
     qr: true,
   },
   theme: {
-    darkMode: undefined,
+    darkMode: true,
```

Comment:
Dark mode will remain unchecked until you toggle it. The persisted slice will currently be `undefined` in your `localStorage`.